### DISCUSSION TITLE
How can I scale up a rendered object on Android?

### DISCUSSION THREAD

**Discussion by maudzung** [google/filament#5671] [Q&A] [answered]
I would like to implement the **zoom in/zoom out** features on Android by touching 2 figures. Can you please share with me examples of using the `filament` lib to do it? 
Thank you so much!

**Comment by bejado** [comment, marked as answer]
I believe our [`GestureDetector.kt`](https://github.com/google/filament/blob/main/android/filament-utils-android/src/main/java/com/google/android/filament/utils/GestureDetector.kt) class has an implementation of 2-fingered zoom.

**Reply by maudzung** [reply]
Thanks for your help.
